browser/ui/views/frame/brave_browser_root_view.cc

```diff
@@ -15,6 +15,9 @@ BraveBrowserRootView::BraveBrowserRootView(BrowserView* browser_view,
 BraveBrowserRootView::~BraveBrowserRootView() = default;
 
 bool BraveBrowserRootView::OnMouseWheel(const ui::MouseWheelEvent& event) {
+  if (!base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs))
```

Comment:
Even in horizontal tabs this annoys me.

Reply:
Yup, I can take it. Here I filed an issue for it. https://github.com/brave/brave-browser/issues/27694